magefiles/magefile.go

```diff
@@ -481,13 +569,13 @@ func Logs() {
 
 // Ensure operator-sdk is installed.
 func EnsureOperatorSDK() {
-	const version = "v1.3.0"
+	const version = "v1.19.0"
 
 	if runtime.GOOS == "windows" {
 		mgx.Must(errors.New("Sorry, OperatorSDK does not support Windows. In order to contribute to this repository, you will need to use WSL."))
 	}
 
-	url := "https://github.com/operator-framework/operator-sdk/releases/{{.VERSION}}/download/operator-sdk_{{.GOOS}}_{{.GOARCH}}"
+	url := "https://github.com/operator-framework/operator-sdk/releases/download/{{.VERSION}}/operator-sdk_{{.GOOS}}_{{.GOARCH}}"
```

Comment:
Sorry about that, it must have never worked?

Reply:
I think this was a recent change.  It worked before.